backend/llm/providers/openai.py

```
@@ -70,6 +74,12 @@ async def chat_completion(
         if tools:
             payload["tools"] = self._convert_tools_to_provider_format(tools)
 
+        # OpenRouter固有の設定
+        if self.is_openrouter and not self.enable_web_search:
+            # Web検索を無効化（transformsを空にする）
```

Ruff (0.14.10)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)